src/utils/viewer/resources.conf

```
@@ -1,4 +1,4 @@
 group=fonts
 
 [file]
-filename=ProggyClean.ttf
+filename=../../../data/fonts/ProggyClean.ttf
```

To avoid having to hardcode the ugly relative path in `viewer.cpp`, add an alias here ([docs](https://doc.magnum.graphics/corrade/classCorrade_1_1Utility_1_1Resource.html#Utility-Resource-compilation)):

```suggestion
filename=../../../data/fonts/ProggyClean.ttf
alias=ProggyClean.ttf
```

---

oo that's slick. Thanks, I wasn't exactly sure how the resources.conf file worked